src/client/components/Marker.js

```diff
@@ -12,4 +13,10 @@ const MapMarker = ({ position, children }) => (
     {children}
   </Marker>
 );
+
+MapMarker.propTypes = {
+  position: PropTypes.array,
+  children: PropTypes.object
```

Comment:
Here I would normally ask you to test a level deeper, but I think we are going to change this component quite a bit soon, so stick with what we have here 👍